<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectrodeCoilSetupDialog</class>
 <widget class="QWidget" name="ElectrodeCoilSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>752</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ElectrodeCoilSetupDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Electrode Sponge">
    <attribute name="title">
     <string>Electrode/Coil Setup</string>
    </attribute>
    <widget class="QTableWidget" name="electrode_coil_tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>271</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="AllInputsTDCS_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All inputs are tDCS electrodes</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ProtoTypeInputCheckbox_">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Only use this prototype input:</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="ProtoTypeInputComboBox_">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>???</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="electrodethicknessCheckBox_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Electrode-sponge thickness:</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="electrodethicknessSpinBox_">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>69</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="invertNormalsCheckBox_">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="geometry">   
    <rect>
     <x>390</x>
     <y>360</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Invert normals to position electrodes on scalp</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ProtoTypeInputCheckbox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>ProtoTypeInputComboBox_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>electrodethicknessCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>electrodethicknessSpinBox_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
